src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Ccda/ProgressNote/PROBLEMS_in_Empty_C-CDA_2.1-C-CDAR2.1-expected.json

```diff
@@ -451,28 +449,6 @@
         "url": "Practitioner/94866a00-1f48-91e8-9327-cc1da205e69b"
       }
     },
```

Comment:
Not generating encounter any more?

Reply:
I add a condition that if one of the required properties is missing in sample data, then the resource will not be generated.  In this case, the required property `class` of `encounter` is missing in this sample data, so the encounter isn't generated.
Do you think is it appropriate to add this condition or just generate the `encounter` resource anyway?

Reply:
I undo it now. The `encounter` is generated and the required property `class` may be missing, which couldn't pass FHIR validator.